packages/entities/entities-plugins/src/components/free-form/shared/KeyValueField.vue

```diff
@@ -43,16 +45,20 @@
       class="ff-kv-field-entry"
     >
       <KInput
+        :id="`${field.path.value}.key`"
         v-model.trim="entry.key"
         class="ff-kv-field-entry-key"
         :data-key-input="index"
+        :data-testid="`${field.path.value}.key`"
```

Comment:
We should adopt a consistent naming convention for these `testid`s.

`{part}-{path}` or `{path}.{part}`? Maybe the former one is better as the latter may make `part` looks like a part of the `path`.